Dockerfile

```diff
@@ -1 +1,27 @@
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base_java11:latest
+RUN useradd -r -g root tomcat
+
+ENV TOMCAT_VERSION=9.0.22  \
+    TOMCAT_MAJOR_VERSION=9
+
+ADD  https://archive.apache.org/dist/tomcat/tomcat-${TOMCAT_MAJOR_VERSION}/v${TOMCAT_VERSION}/bin/apache-tomcat-${TOMCAT_VERSION}.tar.gz /tmp
+
+RUN cd /tmp && \
+    tar -xf apache-tomcat-${TOMCAT_VERSION}.tar.gz && \
+    rm apache-tomcat-${TOMCAT_VERSION}.tar.gz && \
+    mv apache-tomcat-${TOMCAT_VERSION} /opt/tomcat/ && \
+    chown -R tomcat: /opt/tomcat && \
```

Comment:
Same comment as before: All Tomcat files should be owned by `root:root` by default, and readable by the `tomcat` user. If Tomcat absolutely needs to be able to write to some files/directories, then only for those files/directories we can be less restrictive.

Reply:
@naveen-armedia  Wherever Tomcat requires write access, you can provide it so. I am quite sure that Tomcat does not require write access for all files and directories under `/opt/tomcat`. For example the `README.md` won't be touched by Tomcat. This is a trivial example but hopefully you understand my point.

Please change the ownership of all files and directories under `/opt/tomcat` to `root:root`, and give write access to the `tomcat` user only for those files and directories that Tomcat absolutely need to write to. Thanks.